script/Ciel/Day 3/03_01_CIEL03_3_3B.txt

```diff
@@ -392,17 +415,19 @@ A different kind of heartache.
 [sha:0774d560cbe46f5e5481f6a54ca62281ad347459]{
 -- Page 219, Offset 5416.
 -- 先輩の気遣いに心が折れそうで、うつむいて息を吐く。
-Senpai's concern seems to break my heart, so I turned my head and exhaled.
+My heart feels like it's about to break under the weight of Senpai's care,
```

Comment:
"weight of Senpai's care" sounds a little off. "weight of Senpai's worry" maybe? I know "worry" is just god damn everywhere so "weight of Senpai's concern" or consideration?

Reply:
concern sounds good to me